runtime/acala/src/lib.rs

```diff
@@ -1554,6 +1554,32 @@ impl module_homa::Config for Runtime {
 	type RelayChainBlockNumber = RelaychainDataProvider<Runtime>;
 	type XcmInterface = XcmInterface;
 	type WeightInfo = weights::module_homa::WeightInfo<Runtime>;
+	type NominationsProvider = HomaValidatorList;
+}
+
+parameter_types! {
+	pub MinBondAmount: Balance = 10 * dollar(LDOT);
+	pub const ValidatorBackingBondingDuration: BlockNumber = 28 * DAYS;
+	pub ValidatorInsuranceThreshold: Balance = 100_000 * dollar(LDOT);
+	pub const MaxNominations: u32 = 24;
```

Comment:
```suggestion
	pub const MaxNominations: u32 = 16;
```
only 16 for polkadot